### DISCUSSION TITLE
react-native-slider new architecture adoption

### DISCUSSION THREAD

**Discussion by michalchudziak** [reactwg/react-native-new-architecture#38] [Libraries]
Hey! We will be migrating React Native Slider to the new architecture :) Here's tracking issue: https://github.com/callstack/react-native-slider/issues/380

**Comment by cipolleschi** [comment]
Hi @michalchudziak! How is this migration going? Are there any blockers?

**Reply by thymikee** [reply]
@cipolleschi iOS and Android are now merged into the repo and we're close to introducing Fabric in a minor release

**Reply by michalchudziak** [reply]
Here's a [PR](https://github.com/callstack/react-native-slider/pull/410/files). We need a feedback loop, before the actual merge. It should be landed soon tho.

**Comment by thymikee** [comment]
Done & released in: https://github.com/callstack/react-native-slider/releases/tag/v4.3.0

**Reply by cipolleschi** [reply]
Wonderful! Thank you so much!

**Comment by intergalacticspacehighway** [comment]
@thymikee does android require any additional config? I am on react native `0.69.4` and trying slider `4.3.0`. It's working well on iOS but getting below on android. `RNCSlider is not Fabric compatible yet`

<img width="323" alt="Screenshot 2022-08-15 at 6 14 48 PM" src="https://user-images.githubusercontent.com/23293248/184637309-406c4848-73f4-4f68-aa6b-b7523a4804e1.png">

**Reply by cortinico** [reply]
> RNCSlider is not Fabric compatible yet

@thymikee should have more insights here, but I believe the case is that the library is not properly linked on Android (especially as autolinking will be shipped on 0.70 for New Architecture libraries).

**Reply by MateWW** [reply]
You can look at the `Test plan for RN <0.70:` section in the PR description.
There is a step-by-step instruction on how to link it properly
https://github.com/callstack/react-native-slider/pull/410
